Lesson-3/assignment/yours.sol

```diff
@@ -0,0 +1,171 @@
+/*作业请提交在这个目录下*/
+
+=========================================================================================
+第一题：完成今天所开发的合约产品化内容，使用Remix调用每一个函数，提交函数调用截图                    |
+=========================================================================================
+
+https://github.com/yyssjj33/Team-F/raw/53-%E6%9D%A8%E6%A5%AB/Lesson-3/assignment/%E4%BD%9C%E4%B8%9A%E4%B8%89%E7%AC%AC%E4%B8%80%E9%A2%98%E6%88%AA%E5%9B%BE.zip
+
+
+
+=========================================================================================
+第二题：增加 changePaymentAddress 函数，更改员工的薪水支付地址，思考一下能否使用modifier整合某个功能|
+=========================================================================================
+可以整合partialPaid函数到modifier
+modifier partialPaid(employeeId) {
```

Comment:
using partialPay as modifier is interesting. try to add employeeNonExist as well